integration-test/test-pages/broker-protection/pages/form.html

```diff
@@ -45,6 +45,13 @@
     document.forms.testform.onsubmit = (e) => {
       e.preventDefault();
     }
+    setTimeout(() => {
+        document.body.classList.add('delay-complete')
+        const next = document.createElement('div')
+        next.classList.add('slow-element');
+        next.textContent = 'Slow loading element'
+        document.forms.testform.appendChild(next)
+    }, 1000)
```

Comment:
just making changes after 1second, so that the integration test is as real as possible